recipes/yasm/all/conandata.yml

```diff
@@ -1,6 +1,9 @@
 sources:
   "1.3.0":
-    - url: "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz"
+    - url: [
+      "http://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
```

Comment:
```suggestion
      "https://www.tortall.net/projects/yasm/releases/yasm-1.3.0.tar.gz",
```

Reply:
I'm going to commit this to the PR, I cannot check that it works though.